packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Style the Okta-hosted Sign-In Widget
+---
+
+You can add any HTML, CSS, or JavaScript to the **Customization** page that you want. You can also customize the sign-in page per application and with multiple brands.
+
+> **Note:** Before you can get started customizing the Okta-hosted sign-in page, you must have already customized your [Okta URL domain](/docs/guides/custom-url-domain/).
+
+## Edit the sign-in page
+
+The **Customize Sign-In** page offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, from the Developer Console select **Customization**, and then **Signin Page**.
+
+### Change the Okta Sign-In Widget version
+
+The Okta-hosted sign-in page uses the [Sign-In Widget](https://github.com/okta/okta-signin-widget) component to interact with the user. Changing the Okta Sign-In Widget version allows you to leverage or omit specific Sign-In Widget capabilities by specifying a particular version. If you aren't familiar with the Okta Sign-In Widget, Okta recommends that you select the highest **Major Version** and the latest **Minor Version** (default). For details about the Okta Sign-In Widget capabilities that are supported by major and minor versions, see the [GitHub releases page](https://github.com/okta/okta-signin-widget/releases).
+
+1. To make changes to the major and minor versions, select **Edit** in the **Okta Sign-In Widget Version** section header.
```

Comment:
I don't see an edit button in my developer org. Is this because you have to configure a custom domain first? If so, we might want to say that.

Reply:
there's a note at the top of the page for that.